webpack/assets/javascripts/react_app/components/common/Fade.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import helpers from '../../common/helpers';
+
+export default class Fade extends React.Component {
```

Comment:
Correct me if i'm wrong but this doesn't actually fade, just removes the content when time is out?